sherpa/utils/__init__.py

```diff
@@ -1833,27 +1833,27 @@ def multit_pdf(x, mu, sigma, dof):
 
     """
     n = float(dof)
-    x = numpy.asarray(x)
-    mu = numpy.asarray(mu)
-    sigma = numpy.asarray(sigma)
+    x = np.asarray(x)
+    mu = np.asarray(mu)
+    sigma = np.asarray(sigma)
 
     if x.size != mu.size:
         raise TypeError("x and mu sizes do not match")
     if mu.size != sigma.diagonal().size:
         raise TypeError("sigma shape does not match x")
-    if numpy.min(numpy.linalg.eigvalsh(sigma)) <= 0:
+    if np.min(np.linalg.eigvalsh(sigma)) <= 0:
         raise ValueError("sigma is not positive definite")
-    if numpy.max(numpy.abs(sigma - sigma.T)) >= 1.e-9:
+    if np.max(np.abs(sigma - sigma.T)) >= 1.e-9:
         raise ValueError("sigma is not symmetric")
 
     rank = mu.size
     np = float(n + rank)
     coeff = (gamma(np / 2.0) /
-             (gamma(n / 2.0) * numpy.power(n, rank / 2.0) *
-                 numpy.power(numpy.pi, rank / 2.0) *
-                 numpy.sqrt(numpy.abs(numpy.linalg.det(sigma)))))
-    xmu = numpy.mat(x - mu)
-    invsigma = numpy.mat(numpy.linalg.inv(sigma))
+             (gamma(n / 2.0) * np.power(n, rank / 2.0) *
+                 np.power(np.pi, rank / 2.0) *
+                 np.sqrt(np.abs(np.linalg.det(sigma)))))
+    xmu = np.asmatrix(x - mu)
+    invsigma = np.asmattix(np.linalg.inv(sigma))
```

Comment:
same here about matrix already slated for removal. MY feeling is we should get rid of it, while we touch it anyway.